docs/contributing/contributing-code/contributing-code-prerequisites/README.md

```diff
@@ -1,16 +1,49 @@
 # Repository Prerequisites
 
-This page lists the prerequisites for working with the repository. Most contributors should start with the basic prerequisites. Depending on the task you need to perform, you may need to install more tools.
+This page lists the prerequisites for working with the repository. In general, you have three options to get your development setup up and running:
+
+- Remotely using GitHub Codespaces
+- Locally using Visual Studio Code and dev containers leveraging
+- Locally by installing all prerequisites on your
+
+We will walk you through the options in the following sections.
+
+> 📝 **Tip** - We recommend the usage of GitHub Codespaces of local dev containers as they are the most convenient way to get started.
```

Comment:
```suggestion
> 📝 **Tip** - We recommend the usage of GitHub Codespaces or Dev Containers in VS Code as they are the most convenient way to get started.
```

I want to avoid the word `local` because the the VS Code team describes devcontainers as **remote development*. I expect you're trying to clarify the difference between codespaces and devcontainers, but I expect `local` might cause confusion.